lib/awful/layout/init.lua

```diff
@@ -109,6 +110,56 @@ function layout.set(_layout, t)
     tag.setproperty(t, "layout", _layout)
 end
 
+--- Get the layout parameters used for the screen
+--
+-- This should give the same result as "arrange", but without the "geometries"
+-- paramater, as this is computed during arranging
+--
+-- If "t" is given, "screen" is ignored, if none are given, the mouse screen is
+-- used
```

Comment:
Please use "`" instead of normal quotes when referring to arguments, and `@tparam` instead of `@param`.
Also, punctuation.